config/locales/sufia.en.yml

```diff
@@ -212,6 +212,12 @@ en:
       edit:
         header: Edit Work
         in_collections: This Work in Collections
+        additional_fields: "Additional fields"
+        tab:
+          metadata:      "Description"
```

Comment:
I believe this used to be `Descriptions`. Intentional or unintentional change? (If the former, it's OK, but I wanted to be sure.)